javascript/calcs/equip/objects/equip_buffs_list.js

```diff
@@ -733,7 +733,7 @@ equip_buff_list = [
     { "id": 220, "name": "Armor Break", "icon": "armor_break", "category": "debuff", "mode": "mult_red", "options": ["Basic"], "stats": ["pdefreduction"], "values": [[30]], "pvp_values": [[20]] },
     { "id": 572, "name": "Armor Break - Force", "icon": "armor_break", "category": "debuff", "mode": "mult_red", "options": ["Force"], "stats": ["pdefreduction"], "values": [[50]], "pvp_values": [[33]] },
     { "id": 570, "name": "Armor Crash", "icon": "armor_crash", "category": "debuff", "mode": "mult_red", "options": ["Basic", "Killing Blow 2"], "stats": ["pdefreduction"], "values": [[25], [32.5]], "pvp_values": [[20], [26]] },
-    { "id": 221, "name": "Power Break", "icon": ["power_break", "power_break", "persistance", "ultimate_spearmanship", "raging_wave", "giga_blast", "g1_corona"], "category": "debuff", "mode": "mult_red", "options": ["Power Break", "Power Break - Killing Blow 2", "Persistence", "Ultimate Spearmanship", "Tiger Stance 4: Raging Wave", "Giga Blast", "[Force] G-1 Corona (A)"], "stats": ["pdefreduction", "mdefreduction"], "values": [[20, 20], [24, 24], [20, 20], [20, 20], [30, 30], [30, 30], [30, 30]], "pvp_values": [[15, 15], [18, 18], [10, 10], [10, 10], [30, 30], [30, 30], [30, 30]] },
+    { "id": 221, "name": "Power Break", "icon": ["power_break", "power_break", "persistance", "ultimate_spearmanship", "raging_wave", "giga_blast", "g1_corona"], "category": "debuff", "mode": "mult_red", "options": ["Power Break", "Power Break - Killing Blow 2", "Persistence", "Ultimate Spearmanship", "Tiger Stance 4: Raging Wave", "Giga Blast", "[Force] G-1 Corona (A)"], "stats": ["pdefreduction", "mdefreduction","dmg_mod"], "values": [[20, 20, 0], [24, 24, 0], [5, 5, 15], [20, 20, 0], [30, 30, 0], [30, 30, 0], [30, 30, 0]], "pvp_values": [[15, 15, 0], [18, 18, 0], [2.5, 2.5, 7.5], [10, 10, 0], [30, 30, 0], [30, 30, 0], [30, 30, 0]] },
```

Comment:
This is confusing to me? should the change be to Ultimate Spearmanship, also received dmg should be awk_dmg_mod probably so it is correctly additive to awakening? https://elwiki.net/w/Ultimate_Spearmanship
and Persistence removed completely? https://elwiki.net/w/Persistence